spec/models/work_packages/scopes/allowed_to_spec.rb

```diff
@@ -132,7 +138,22 @@
         subject { described_class.allowed_to(user, action) }
 
         it "returns the authorized work package" do
-          expect(subject).to contain_exactly(work_package_in_private_project)
+          expect(subject).to include(work_package_in_private_project)
```

Comment:
this line should not have been changed. It should be `contain_exactly` so that we test that the other wp from same project is not visible.